ci/docsgen/schema.py

```diff
@@ -126,12 +127,16 @@ def parse_fmarkdown_schema(raw_schema: dict,
   schema, sample = get_configuration_schema(commit=commit_sha,
                                             raw_schema=(raw_schema := {}),
                                             get_sample=True)
+  schema_meta = OrderedDict(**{
+    '#Revision': {
+      'OpenCore-Version': entry.get('version', 'Unknown'),
+    }
+  })
```

Comment:
Schema docs will need to be updated on a regular cadence following each OpenCore release. Pre-release builds of OpenCore will also need to be smoke-tested against any schema validation in case new schema entries are added that must be specially handled.

Additionally, the usage of the '#Revision' dict mimics the same usage by OCLP in its payload config.plist [here](https://github.com/dortania/OpenCore-Legacy-Patcher/blob/main/payloads/Config/config.plist#L5-L13). Special care must be taken when introducing metadata keys that may conflict with other projects.